impeller/tools/build_metal_library.py

```diff
@@ -93,6 +93,11 @@ def main():
     command += [
         '--std=macos-metal1.2',
     ]
+  elif args.platform == 'ios':
+    command += [
+        '--std=ios-metal1.2',
```

Comment:
It looks like these will get duplicated when `args.optimize` is true.

Reply:
Whoops, this wasn't an intentional change, removed. It's a diff I lug around in my git stage to stop the xcode GPU profiler from crashing in debug unopt.